swiftlint.yml

```diff
@@ -85,7 +85,37 @@ cyclomatic_complexity:
 
 custom_rules:
   combine_cancellable_set:
+    # Matches:
+    # - [AnyCancellable]
     name: "AnyCancellables as Set"
     regex: "\\[AnyCancellable\\]"
     message: "Combine Cancellables must be put in a Set rather than Array."
     severity: error
+  direct_date_usage:
+    # Matches:
+    # - Date()
+    # - Date.init()
+    # - Date.now
+    # - Date.timeIntervalSinceReferenceDate
+    # - Date.init(timeIntervalSinceNow: 123)
+    # - Date(timeIntervalSinceNow: 123)
+    # - NSDate.now
+    # - ,NSDate.now
+    # - ,Date()
+    #
+    # Does not match:
+    # - Date.timeIntervalBetween1970AndReferenceDate
```

Comment:
Why this should not match, and why do you even mention it?

Reply:
😄 😄 😄 Don't know, really 😄  Should I remove it?